### PR TITLE
Create the home banking links and a new style for name

### PR DESCRIPTION
**Pull request by SantiDepetris**
Url for homebankings added
https://github.com/eldolar/web/issues/5

### CODE REVIEW

File: src/components/Bank.js [hunk 1]
```diff
@@ -2,12 +2,14 @@ import React from 'react';
...
 
-export default function Bank({ bank: { name, sell, buy } }) {
+export default function Bank({ bank: { name, sell, buy, url } }) {
   const noData = hasExchange(sell, buy) ? '' : `${styles.no_data}`;
   const mainClasses = `${styles.box} ${noData}`;
   return (
     <div className={mainClasses} data-test='bank-box'>
-      <h1 className={styles.name}>{name}</h1>
+      <h1 className={styles.name} onClick={ () => window.open(url, '_blank') }>
+      {name}
```
Comment (edudepetris) [inline src/components/Bank.js]: ident with 2 spaces.

Comment (edudepetris) [reply src/components/Bank.js]: I like your idea, don't change it 👍 

As alternative you can use a `<a>`

```jsx
<h1 ... >
  <a href={url}>{name}
```

File: src/styles/Bank.module.scss [hunk 2]
```diff
@@ -17,6 +17,11 @@
   color: $text;
   text-align: center;
   margin-bottom: 0;
+
+  &:hover {
+    color: blue;
```
Comment (edudepetris) [inline src/styles/Bank.module.scss]: what if we use another color like #C969CD
|#C969CD|
|-|
|Screen Shot 2020-10-23 at 8 23 03 PM|

**Review by edudepetris — CHANGES_REQUESTED** [review]
some tweaks!

File: src/components/Bank.js (hunk 1 above)
Comment (edudepetris) [inline src/components/Bank.js]: @SantiDepetris ident with 2 spaces

Comment (SantiDepetris) [reply src/components/Bank.js]: Done

**Review by edudepetris — APPROVED** [review]

**Pull request merged**